examples/shapes/shapes_recursive_tree.c

```diff
@@ -0,0 +1,138 @@
+        branches[count++] = (Branch){start, initialEnd, 0.0f, length};
+
+        for (int i = 0; i < count; i++) {
+            Branch branch = branches[i];
+            if (branch.length < 2) continue;
+
+            float nextLength = branch.length * branchDecay;
+
+            if (count < maxBranches && nextLength >= 2) {
+                Vector2 branchStart = branch.end;
+
+                Vector2 branchEnd1 = CalculateBranchEnd(branchStart, branch.angle + theta, nextLength);
+                Vector2 branchEnd2 = CalculateBranchEnd(branchStart, branch.angle - theta, nextLength);
+
+                branches[count++] = (Branch){branchStart, branchEnd1, branch.angle + theta, nextLength};
+                branches[count++] = (Branch){branchStart, branchEnd2, branch.angle - theta, nextLength};
+            }
+        }
+
+        // Draw
+        //----------------------------------------------------------------------------------
+        BeginDrawing();
+
+            ClearBackground(RAYWHITE);
+
+            for (int i = 0; i < count; i++) {
+                Branch branch = branches[i];
+                if (branch.length >= 2) {
+                    if (!bezier) DrawLineEx(branch.start, branch.end, thick, RED);
+                    else DrawLineBezier(branch.start, branch.end, thick, RED);
+                }
+            }
+
+            DrawLine(580, 0, 580, GetScreenHeight(), (Color){ 218, 218, 218, 255 });
+            DrawRectangle(580, 0, GetScreenWidth(), GetScreenHeight(), (Color){ 232, 232, 232, 255 });
+
+            // Draw GUI controls
+            //------------------------------------------------------------------------------
+            GuiSliderBar((Rectangle){ 640, 40, 120, 20}, "Angle", TextFormat("%.0f", angle), &angle, 0, 180);
+            GuiSliderBar((Rectangle){ 640, 70, 120, 20 }, "Length", TextFormat("%.0f", length), &length, 12.0f, 240.0f);
+            GuiSliderBar((Rectangle){ 640, 100, 120, 20}, "Decay", TextFormat("%.2f", branchDecay), &branchDecay, 0.1f, 0.78f);
+            GuiSliderBar((Rectangle){ 640, 130, 120, 20 }, "Depth", TextFormat("%.0f", treeDepth), &treeDepth, 1.0f, 12.f);
+            GuiSliderBar((Rectangle){ 640, 160, 120, 20}, "Thick", TextFormat("%.0f", thick), &thick, 1, 8);
+            GuiCheckBox((Rectangle){ 640, 190, 20, 20 }, "Bezier", &bezier);
+            //------------------------------------------------------------------------------
+
+            DrawFPS(10, 10);
+
+        EndDrawing();
+        //----------------------------------------------------------------------------------
+    }
+
+    // De-Initialization
+    //--------------------------------------------------------------------------------------
+    CloseWindow();        // Close window and OpenGL context
+    //--------------------------------------------------------------------------------------
+
+    return 0;
+}
+
+static Vector2 CalculateBranchEnd(Vector2 start, float angle, float length) {
```

Comment:
Please, could you review it to follow raylib code conventions with aligned brackets?